src/main/java/cobook/buddywisdom/cancellation/service/CancelRequestService.java

```diff
@@ -0,0 +1,78 @@
+package cobook.buddywisdom.cancellation.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import cobook.buddywisdom.cancellation.controller.DirectionType;
+import cobook.buddywisdom.cancellation.domain.CancelRequest;
+import cobook.buddywisdom.cancellation.dto.response.CancelRequestResponseDto;
+import cobook.buddywisdom.cancellation.exception.ConfirmedCancelRequestException;
+import cobook.buddywisdom.cancellation.exception.NotFoundCancelRequestException;
+import cobook.buddywisdom.cancellation.mapper.CancelRequestMapper;
+import cobook.buddywisdom.coach.domain.CoachSchedule;
+import cobook.buddywisdom.coach.service.CoachScheduleService;
+import cobook.buddywisdom.global.exception.ErrorMessage;
+import cobook.buddywisdom.mentee.service.MenteeScheduleService;
+
+@Service
+@Transactional(readOnly = true)
+public class CancelRequestService {
+	private final CancelRequestMapper cancelRequestMapper;
+	private final CoachScheduleService coachScheduleService;
+	private final MenteeScheduleService menteeScheduleService;
+
+	public CancelRequestService(CancelRequestMapper cancelRequestMapper, CoachScheduleService coachScheduleService,
+		MenteeScheduleService menteeScheduleService) {
+		this.cancelRequestMapper = cancelRequestMapper;
+		this.coachScheduleService = coachScheduleService;
+		this.menteeScheduleService = menteeScheduleService;
+	}
+
+	public List<CancelRequestResponseDto> getCancelRequest(long memberId, DirectionType direction) {
+		List<CancelRequest> cancelRequestList = new ArrayList<>();
+
+		if (direction.equals(DirectionType.SENT)) {
+			cancelRequestList = cancelRequestMapper.findBySenderId(memberId);
+		}
+		if (direction.equals(DirectionType.RECEIVED)) {
```

Comment:
```suggestion
		} else if (direction.equals(DirectionType.RECEIVED)) {
```

direction이 `SENT`라면 아래 조건을 검사하지 않아도 될 것 같아요

Reply:
넵!